guides/user/plugins/index.md

```diff
@@ -200,6 +202,33 @@ spinnaker:
 
 Note: `hal plugins enable` and `hal plugins disable` enable or disable _all_ plugins, so use with caution.
 
+### Plugin configuration without Halyard
+
+To avoid each service restarting and downloading the plugin, _do not_ add the plugin using Halyard. Instead, configure the plugin in the service's local file. For example, if your plugin extends Orca, add configuration to your `orca-local.yml` file.
+
+```yaml
+spinnaker:
+  extensibility:
+    plugins:
+      <unique-plugin-id>:
+        id: <unique-plugin-id>
+        enabled: <true-false>
+        version: <version>
+        extensions:
+          <extension-name>:
+            id: <extension-name>
+            enabled: <true-false>
+            config: {}
+```
+
+The plugin developer should provide configuration details in YAML format. If not:
+
+1. Add the plugin using Halyard
```

Comment:
Periods at the ends of each of these

Reply:
Bullet lists with periods look so wrong to me. I must be defaulting to what I learned in grade school English class in the 1970s (still remember diagramming sentences too).  However, since both the _Chicago Manual of Style_ and the _Google developer documentation style guide_ call for periods if the bullet point is a complete sentence, I can't really argue, can I?

Reply:
Yeah, that's right. 😁 
Numbered lists too.
I have a strong feeling toward getting rid of periods at the ends of list items like this. The period's just a delimited, and there's already the line break and bullet to delimit the phrases or sentences. BUT keeping the period when these are complete sentences is the accepted style out there. One advantage of doing it this way is that if we want to add a sentence to an item in a list like this, we can do so without now making the list less parallel. (Though I like to add those as list paragraphs, between the bullet or number items)